quotes.txt

```diff
@@ -14,3 +14,39 @@ Windows is "I'm locking my sandwich in this safe. Why are you
 worried it will go stale?"
 
     --  pjf, IRC
+%
+Jenkins is refusing to work a simple fix and keeps reporting the
+build as broken. 3 different people are trying to figure out why
+NetKAN reports an error and calls a forth one to come check it out.
+
+"licence" is misspelled
+
+    -- Daz NetKAN#3159
+%
+A PR to break RealSolarSystem into multiple pieces is opened and
+then goes on for two months of back and forth, commiting and testing.
+A new version of KSP is released and RSS switches engine.
+
+So this is funny and sad at the same time but seeing as RSS is switching over to Kopernicus all of this was for naught and is no longer needed.
+
+    -- Daz NetKAN#889
+%
+One individual, one goal, many _MANY_ PRs
+
+Homepage added
+Adding homepage
+Added homepage
+
+    -- plague006 all early PRs
+%
+<ChucklesTheBeard> ayy
+<p6-away> hey boss
+<pjf> Yes. Chuckles is boss now. This pleases me. :)
+
+    -- pjf on irc
+%
+Sometimes you just don't want to fix things.
```

Comment:
That was my plugin that sets me in away status when my computer shuts down or sleepmode.